test/e2e/framework/deploy/deploy.go

```diff
@@ -0,0 +1,115 @@
+// +build e2e
+
+package deploy
+
+import (
+	"fmt"
+	"io/ioutil"
```

Comment:
`io/ioutil` is deprecated in go 1.16: https://golang.org/doc/go1.16#ioutil. Let's use `os` package.

Reply:
removed